cmd/gateway.go

```diff
@@ -25,7 +25,7 @@ func init() {
 	gatewayCmd.Flags().StringVarP(&version_manager.VersionManagerEndpoint, "version-manager", "", version_manager.DefaultVersionManagerEndpoint, "GRPC Endpoint for Version Manager Service")
 	gatewayCmd.Flags().StringVarP(&storage.StorageServiceEndpoint, "storage", "", storage.DefaultStorageServiceDefaultEndpoint, "GRPC Endpoint for Storage Service")
 	gatewayCmd.Flags().StringVarP(&tag_manager.TagManagerEndpoint, "tag-manager", "", tag_manager.DefaultTagManagerEndpoint, "GRPC Endpoint for Tag Service")
-
+	//gatewayCmd.Flags().StringVarP(&release.ReleaseEndpoint, "tag", "", release.DefaultReleaseEndpoint, "GRPC Endpoint for Tag Service")
```

Comment:
why is it commented out?

Reply:
Commented out for the time being, as it's not yet used in the gateway (or browse) service. We can uncomment. Thanks.